content/applications/finance/accounting/receivables/customer_payments/followup.rst

```diff
@@ -14,31 +14,42 @@ Configuration
 =============
 
 To configure a :guilabel:`Follow-Up Action`, go to :menuselection:`Accounting --> Configuration -->
-Follow-up Levels`. Several follow-up actions are available by default, and you can customize them
-any way you want or create new ones to trigger the following actions:
+Follow-up Levels`. Several follow-up actions are available by default, but you can
+:guilabel:`Create` new ones if required. You can change its **name** and the **number of days**
+after which the follow-up actions are triggered. The follow-up :guilabel:`Actions` available are:
 
-- Send an email;
+- :guilabel:`Send an email`;
 - :ref:`Send an SMS message <pricing/pricing_and_faq>`;
-- Print a letter;
+- :guilabel:`Print a letter`;
 - :ref:`Send a letter <customer_invoices/snailmail>`;
-- Manual action (creates a task).
+- :guilabel:`Manual action` (creates a task).
 
-It is also possible to automatically send a reminder by enabling the :guilabel:`Auto Execute`
-option, within a specific :guilabel:`Follow-Up Action`.
+It is possible to automatically send a reminder by enabling the :guilabel:`Auto Execute` option, and
+attaching the *open* invoice(s) by enabling :guilabel:`Join open invoices`, within a specific
+:guilabel:`Follow-Up Action`.
 
+Depending on the :guilabel:`Actions` enabled, the :guilabel:`Message` tab contains different message
+options. The :guilabel:`Email Subject` and the :guilabel:`Description` or content of the email
+automatically used. If you selected :guilabel:`SMS`, the content of the SMS text. In case of
+:guilabel:`Manual Action`, you can :guilabel:`Assign a Responsible` person to manage that follow-up
+and the :guilabel:`Manual Action Type` (:guilabel:`Email`, :guilabel:`Call`, etc.), as well as input
+a description of the :guilabel:`Action To Do`.
```

Comment:
This sentence "sounds" a bit weird, especially with the mix of guilabel and words.


```If you enabled :guilabel:`Manual Action`, you can assign someone in the guilabel:`Assign a Responsible` field to manage that follow-up ....```